flytekit/types/schema/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .types import (
     FlyteSchema,
+    FlyteSchemaTransformer,
```

Comment:
Is this necessary?  The transformer shouldn't be exposed to the end user.  The idea of this file was to only import things that the user might want to reference/use.